test/utils/shippable/shippable.sh

```diff
@@ -117,7 +117,7 @@ function cleanup
                     flags="${flags//=/,}"
                     flags="${flags//[^a-zA-Z0-9_,]/_}"
 
-                    bash <(curl -s https://codecov.io/bash) \
+                    bash <(curl -s https://ansible-ci-files.s3.us-east-1.amazonaws.com/ansible-test/codecov) \
```

Comment:
This change should be made in `.azure-pipelines/scripts/publish-codecov.sh` instead.

This part of the script is obsolete.

Reply:
I've merged https://github.com/ansible/ansible/pull/74464 to remove the obsolete code.

Reply:
Oops. I'll make the change in the correct location.

Reply:
One thing I was wondering is whether or not we wanted to add an integrity checking step. It would look something like this:

```
curl -s https://raw.githubusercontent.com/codecov/codecov-bash/1.0.2/SHA256SUM | sed 's/codecov/codecov.sh/' | shasum --check --algorithm 256 --ignore-missing
```

I could put the checksum file in S3 and give it a consistent name so we can update the script and hash without having to touch this file.